diff --git a/dist/ai-sdk.d.ts b/dist/ai-sdk.d.ts
index 83bf679668d2643c54e0a9e3d6445df122678ee7..1cf6e9c46c9f01fd87af5578d2a8032eef415258 100644
--- a/dist/ai-sdk.d.ts
+++ b/dist/ai-sdk.d.ts
@@ -1,3 +1,5 @@
-import type { LanguageModelV2 } from "@ai-sdk/provider";
+import type { LanguageModelV2, LanguageModelV3 } from "@ai-sdk/provider";
 export declare const traceAISDKModel: (model: LanguageModelV2) => LanguageModelV2;
+
+export declare const traceAISDKModelV3: (model: LanguageModelV3) => LanguageModelV3;
 //# sourceMappingURL=ai-sdk.d.ts.map
\ No newline at end of file
